### PR TITLE
Chore: restoring the create_tx.cy E2E file into the smoke folder

### PR DESCRIPTION
**Pull request by DaniSomoza**
## What it solves

Fixes `create_tx.cy.js` E2E test

## How this PR fixes it

It seems that the `estimations` endpoint is not working properly in the CI, and after some workarounds I think that the best option here is just to stub this endpoint. 

At the end we only want to make sure that the transaction modal works as expected

```
    cy.intercept('POST', '**/multisig-transactions/estimations', {
      statusCode: 200,
      body: { currentNonce: currentNonce, recommendedNonce, safeTxGas: '45006' },
    }).as('EstimationRequest')
```    
    
see [cypress docs](https://docs.cypress.io/api/commands/intercept#Syntax) for more info about stubbing request.

### CODE REVIEW

File: cypress/e2e/smoke/create_tx.cy.js [hunk 1]
```diff
@@ -35,17 +36,18 @@ describe('Queue a transaction on 1/N', () => {
...
   })
 
   it('should create a queued transaction', () => {
     // Wait for /estimations response
-    cy.intercept('POST', '/**/multisig-transactions/estimations').as('EstimationRequest')
-
-    cy.contains('Next').click()
+    cy.intercept('POST', '**/multisig-transactions/estimations', {
+      statusCode: 200,
+      body: { currentNonce, recommendedNonce, safeTxGas: '45006' },
```
Comment (katspaugh) [inline cypress/e2e/smoke/create_tx.cy.js:47]: Can you try reverting this and check if it works now? The backend was fixed AFAIK.

Comment (DaniSomoza) [reply cypress/e2e/smoke/create_tx.cy.js:47]: ok sure!

Comment (DaniSomoza) [reply cypress/e2e/smoke/create_tx.cy.js]: It seems that in the first execution `create_tx.cy` test passed but in the second try it failed 😔

https://cloud.cypress.io/projects/exhdra/runs/e5e81fcb-d9cf-46c2-bc7b-e93ba5275027/test-results/14d6de18-41f1-4cab-95f4-44b33584e3ca/video

Comment (DaniSomoza) [reply cypress/e2e/smoke/create_tx.cy.js]: I am re-running it again 🤞

Comment (DaniSomoza) [reply cypress/e2e/smoke/create_tx.cy.js]: `create_tx.cy.js` test [failed](https://cloud.cypress.io/projects/exhdra/runs/74555bdc-d4ef-4687-98ff-d1f0a0fb38a5/test-results/4a73ccbc-ab2c-486b-9e00-3c66c6fb77d5/video) again....

Comment (katspaugh) [reply cypress/e2e/smoke/create_tx.cy.js]: Damn.

**Review by katspaugh — APPROVED** [review]
Looks like it passes! Let's merge it. :)

**Pull request merged**